api/src/pcapi/core/fraud/api/ubble.py

```diff
@@ -68,3 +72,7 @@ def get_ubble_fraud_check(identification_id: str) -> typing.Optional[fraud_model
         .one_or_none()
     )
     return fraud_check
+
+
+def does_match_ubble_test_email(email: str) -> typing.Optional[re.Match]:
```

Comment:
c'est là qu'il faudrait `not settings.IS_PROD` ?

Reply:
yep